config/environments/test.rb

```diff
@@ -45,4 +45,56 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
```

Comment:
I prob. should have done this in requests as well (instead of the `if` statements in `application.rb`).